scripts/merge-measures-data.js

```diff
@@ -47,9 +47,70 @@ function mergeQpp(qppJson) {
       if (performanceRateInfo.metricType) {
         qppJson[index].metricType = performanceRateInfo.metricType;
       }
+
     }
   });
 
   console.error('did not find measure details for the following', measuresNotFound);
+
+  // separate passes for ecqm data to keep things simple
+  // load extracted ecqm data
+  const generatedEcqmStrataJson = JSON.parse(fs.readFileSync(path.join(__dirname, '../util/generated-ecqm-data.json'), 'utf8'));
```

Comment:
Yeah, I started that way, but wanted to keep them separate at first since we could reason about the new additions separately without also considering the original logic in merge-measures-data. For example, I ran into an issue with the guard clause exiting early when no perf rate info was found - I didn't want to change the resulting logic to consider when ecqm data was found but not perf rate data (and vice versa).

I'm thinking that the refactor can explicitly enumerate the different sources of data and the fields pulled from them.